### PR TITLE
upgrade/truffle

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

File: lib/dns.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+var exports = module.exports = {};
```
Comment (Arachnid) [inline lib/dns.js]: Aren't we trying to get rid of this code, not propagate it to more places?

Comment (decanus) [reply lib/dns.js]: yeah, this is a WIP.

File: migrations/2_deploy_contracts.js [hunk 2]
```diff
@@ -21,14 +21,15 @@ module.exports = function(deployer, network) {
             return;
         }
 
-        await deployer.deploy([[ENSRegistry], [DummyDNSSEC]]);
+        await deployer.deploy(ENSRegistry);
```
Comment (Arachnid) [inline migrations/2_deploy_contracts.js]: Any reason for this change?

File: test/Helpers/Utils.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+const packet = require('dns-packet');
```
Comment (Arachnid) [inline test/Helpers/Utils.js]: Didn't you just make a new package for this?

File: migrations/2_deploy_contracts.js (hunk 2 above)
Comment (decanus) [reply migrations/2_deploy_contracts.js]: yes, because of latest truffle. Bundled deploys no longer exist

File: test/Helpers/Utils.js (hunk 3 above)
Comment (decanus) [reply test/Helpers/Utils.js]: That will be taken care of in another pull request.

**Pull request merged**